security/publish_spec.rb

```diff
@@ -3,9 +3,20 @@
 describe 'Publish', type: :feature do
   before :each do
     @conf = read_config
+    @aem_publish = init_publish_client(@conf['aem']['publish'])
   end
 
   it 'should not be able to login using default admin password' do
+    # use aem api calls
+    bundle = @aem_publish.bundle('com.adobe.cq.social.cq-social-forum')
+    begin
+      result = bundle.start
+    rescue RubyAem::Error => error
+      # response should be unauthorized
+      # checking for 500 since a NoAuthenticationHandlerException Java Exception is thrown, thus a server error
```

Comment:
Add a link to the spec https://github.com/shinesolutions/ruby_aem/blob/master/conf/spec.yaml#L49 , and modify the test so that instead of checking for 500, it checks that it's not 200 and not 404 (as per spec).